lib/fastlane/plugin/firebase_app_distribution/client/firebase_app_distribution_api_client.rb

```diff
@@ -110,9 +118,9 @@ def upload_binary(app_id, binary_path, platform)
           request.headers["X-APP-DISTRO-API-CLIENT-VERSION"] = Fastlane::FirebaseAppDistribution::VERSION
         end
       rescue Faraday::ResourceNotFound
-        UI.crash!("#{ErrorMessage::INVALID_APP_ID}: #{app_id}")
+        UI.user_error!("#{ErrorMessage::INVALID_APP_ID}: #{app_id}")
       rescue Errno::ENOENT
```

Comment:
nit: Not 100% clear that `Errno::ENOENT` is raised when the file can't be opened. Consider adding a comment here below to explain the reason for the exception being raised.

Reply:
Good point